ieee.csl

```diff
@@ -234,11 +230,21 @@
           </group>
           <!-- [Online Video]. -->
           <group prefix="[" suffix="]" delimiter=" ">
-            <text term="online" text-case="capitalize-first"/>
             <choose>
-              <if type="motion_picture">
-                <text value="video" text-case="capitalize-first"/>
+              <if variable="medium" match="any">
+                <text variable="medium" text-case="capitalize-first"/>
               </if>
+              <else-if type="speech" variable="genre" match="all">
+                <text variable="medium" text-case="capitalize-first"/>
```

Comment:
This will never render anything because if an item has medium it will hit the previous condition

Reply:
sorry, that was a typo. I meant to use genre, but ended up removing this again because "type" is used for things that definitely shouldn't be in square brackets for IEEE.